tests/test_handler_wandb_stats.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import glob
+import os
+import shutil
+import tempfile
+import unittest
+
+from ignite.engine import Engine
+
+from monai.handlers import WandbStatsHandler
+from monai.utils import optional_import
+
+wandb, _ = optional_import("wandb")
+
+
+@unittest.skipUnless(wandb, "Requires wandb installation")
+class TestWandbStatsHandler(unittest.TestCase):
+    def test_metric_tracking(self):
+        tempdir = tempfile.TemporaryDirectory()
+        os.system("wandb offline")
+        os.environ["WANDB_DIR"] = tempdir.name
+
+        wandb.init(dir=tempdir.name)
```

Comment:
I see the `wandb` is global, is it thread-safe and supports multi-processing?
I want suggest to add some tests to call `wandb.init(dir=tempdir.name)` and the whole handler in multi-thread and multi-processing logic.
For multi-processing test, you can refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_cumulative_average_dist.py
For multi-thread test, you can easily add a test function here, refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/dataset.py#L875-L878
What do you think?

Thanks.